python/ray/data/datasource/parquet_datasource.py

```diff
@@ -36,6 +36,9 @@
 PARQUET_READER_ROW_BATCH_SIZE = 100000
 FILE_READING_RETRY = 8
 
+# The estimated bytes size multiplier for reading Parquet data source in Arrow.
+PARQUET_TO_ARROW_SIZE_MULTIPLIER = 5
```

Comment:
Can we add some explanation here for context (or link to this PR for discussion?)

Reply:
@ericl - sure, added.